retis/src/process/cli/pcap.rs

```diff
@@ -149,6 +200,40 @@ impl EventParser {
                 id
             }
         };
+        Ok(id)
+    }
+
+    /// Parse & process a single Retis event.
+    fn parse(&mut self, event: &mut Event) -> Result<Vec<Block<'_>>> {
+        // Having a common & a kernel section is mandatory for now, seeing a
+        // filtered event w/o one of those is bogus.
+        let common = event
+            .common
+            .as_ref()
+            .ok_or_else(|| anyhow!("No common section in event"))?;
+
+        self.stats.processed += 1;
+
+        let comment = format!(
+            "{}",
+            event.display(&DisplayFormat::new().multiline(true), &FormatterConf::new())
```

Comment:
I'm wondering what is the purpose of embedding the full formatted even in the PCAP entry, in addition to all its JSON fields? Wireshark is already displaying the event in its own way, which is the purpose of this PR. Or is that for easing comparing events between Retis' output and Wireshark (is that a thing?)?

Reply:
Wireshark will display all the event fields so users can inspect each, filter, etc.
But having the "pretty" event printed in the comments felt to me as a good way of quickly getting all the information. It summarizes the information that is otherwise dispersed in the field tree. There is a reason and a lot of care put into how to display the metadata (were not just dumping the json representation in `retis print`), so not having that information feels like a pity.

Reply:
OK, makes sense. If we see at some point all this "duplicated" information is taking too much space we can always add cli arguments to enable/disable some parts.

(Especially with large collections).